ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml

```diff
@@ -11188,16 +11364,6 @@ periodics:
     job-release: "4.16"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-outpost-day2-f28-longduration-cloud
-  reporter_config:
```

Comment:
@sergiordlr I know it is deleted by dptp tool automatically, but it is not correct for dptp.
so, could you please recovery these notifications manually before pushing the PR? thanks

Reply:
Done

Reply:
@kuiwang02 Can you review again, thx